File: c/4278.c
#include <stdio.h>
 
main()
{
    int c, i, t, l,state=0;
    char arr[2][200]={0};
 
    for (i=0;(c=getchar())!='\n';++i)
        arr[0][i]=c;
    for (t=0;(c=getchar())!= EOF;++t)
        arr[1][t]=c;
    for (i=0;arr[0][i]!='\0';++i){
        for (t=0;arr[1][t]!='\0';++t){
            if(arr[0][i]==arr[1][t]){
                l=i;
                state=1;
                break;
            }
        }
        if(state==1)
            break;
    }
    if(state==1)
        printf("%d",l);
    else{
        printf("-1");
        return -1;
    }
}

File: java/4278.java
import java.util.*;
public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
 
        int c, i, t, l=0,state=0;
        char[][] arr = new char[2][200];
 
        arr[0] = sc.nextLine().toCharArray();
        arr[1] = sc.nextLine().toCharArray();
        for (i=0;i<arr[0].length;++i){
            for (t=0;t<arr[1].length;++t){
                if(arr[0][i]==arr[1][t]){
                    l=i;
                    state=1;
                    break;
                }
            }
            if(state==1)
                break;
        }
        if(state==1)
            System.out.printf("%d", l);
        else{
            System.out.printf("-1");
        }
    }
}